app/controllers/SecondContactHavePhoneController.scala

```diff
@@ -46,7 +46,7 @@ class SecondContactHavePhoneController @Inject() (
     with I18nSupport
     with ContactHelper {
 
-  val form = formProvider()
+  val form = formProvider("clientSecondContactHavePhone")
```

Comment:
This should just be "secondContactHavePhone" because this controller is for the organisation journey